### PR TITLE
add state

### PR DESCRIPTION
**Pull request by VictorYarosh**

### CODE REVIEW

File: src/components/ui/navbar/index.tsx [hunk 1]
```diff
@@ -7,8 +7,23 @@ const NavBar = () => {
     <Navbar>
       <Logo />
       <Links>
-        <LinksPage>GitHub</LinksPage>
-        <LinksPage>About</LinksPage>
+        <LinksPage
+          onClick={() =>
+            window.open('https://github.com/VictorYarosh/weather', '_blank')
+          }
+        >
```
Comment (toserg) [inline src/components/ui/navbar/index.tsx]: Why so complicated? 
use just <a href="...." >
This is a simple link, not a button.

File: src/components/ui/navbar/index.tsx [hunk 2]
```diff
@@ -7,8 +7,23 @@ const NavBar = () => {
...
+          GitHub
+        </LinksPage>
+        <LinksPage
+          onClick={() =>
+            window.open(
+              'https://ua.linkedin.com/in/victor-yarosh-801b45212/uk?trk=people-guest_people_search-card',
+              '_blank'
+            )
+          }
+        >
```
Comment (toserg) [inline src/components/ui/navbar/index.tsx]: the same

File: src/components/ui/weather-card-control/weather-card-control.styled.ts [hunk 3]
```diff
@@ -8,22 +8,29 @@ export const AddNewButton = styled.div`
...
+  border-radius: 25px;
+  padding-left: 5px;
+
   input {
     width: 85%;
     height: 30px;
-    margin: 5px;
+    margin: 5px 3px;
     border: none;
+    border-radius: 25px;
```
Comment (toserg) [inline src/components/ui/weather-card-control/weather-card-control.styled.ts]: add some padding, when you type inside first symbol is cut

**Review by toserg — CHANGES_REQUESTED** [review]

File: src/components/ui/navbar/index.tsx (hunk 2 above)
Comment (VictorYarosh) [reply src/components/ui/navbar/index.tsx]: fix this

File: src/components/ui/navbar/index.tsx (hunk 1 above)
Comment (VictorYarosh) [reply src/components/ui/navbar/index.tsx]: fix this

File: src/components/ui/weather-card-control/weather-card-control.styled.ts (hunk 3 above)
Comment (VictorYarosh) [reply src/components/ui/weather-card-control/weather-card-control.styled.ts]: ready this

File: src/components/ui/navbar/index.tsx (hunk 2 above)
Comment (toserg) [reply src/components/ui/navbar/index.tsx]: fixed*
please use the right time

**Review by toserg — APPROVED** [review]

**Pull request merged**